src/main/resources/application.yml

```diff
@@ -264,5 +264,11 @@ app:
     response.timeout: ${RESPONSE_TIMEOUT:60000}
     read.handler.timeout: ${READ_TIMEOUT_HANDLER:60000}
     write.handler.timeout: ${WRITE_TIMEOUT_HANDLER:60000}
-
-
+  email-notification:
+    comma-delimiter: ${$EMAIL_NOTIFICATION_COMMA_DELIMITER:,}
+    base-url: ${EMAIL_NOTIFICATION_BASE_URL:/idpay/email-notification}
+  selc:
+    base-url: ${SELC_BASE_URL:/external/v1}
+    headers:
+      subscription-key: ${SELC_EXTERNAL_API_KEY:}
+      uid: ${SELC_UID:idpay}
```

Comment:
let's verify if we could send whatever string as uid... theoretically, it should be the logged user, but this is a batch...
Thus, SelfCare should ignore this field, but let's test this calling it from your pc...